src/VirtoCommerce.SimpleExportImportModule.Data/Validation/ImportProductPricesExistanceValidator.cs

```diff
@@ -0,0 +1,41 @@
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using FluentValidation;
+using FluentValidation.Validators;
+using VirtoCommerce.PricingModule.Core.Model.Search;
+using VirtoCommerce.PricingModule.Core.Services;
+using VirtoCommerce.SimpleExportImportModule.Core.Models;
+
+namespace VirtoCommerce.SimpleExportImportModule.Data.Validation
+{
+    public class ImportProductPricesExistenceValidator: AbstractValidator<ImportProductPrice[]>
+    {
+        public const string ExistingPrices = nameof(ExistingPrices);
+
+        private readonly IPricingSearchService _pricingSearchService;
+        private readonly bool _not;
```

Comment:
It is a very meaningless field name. Please, try to improve it.

Reply:
Tried to fix